source/reference/mongosync-behavior.txt

```diff
@@ -21,8 +19,13 @@ The ``mongosync`` binary is the primary process used in
 {+c2c-product-name+}. ``mongosync`` migrates data from one cluster to
 another and can keep the clusters in continuous sync. 
 
-This document provides a complete overview of the ``mongosync`` command
-line options.
+This document provides a complete overview of ``mongosync`` behavior. For
+information on how the ``mongosync`` process works, see :ref:`about-mongosync`. 
```

Comment:
[question] Do you think we can cut this line out because the purpose of the page is indicated by the title?

Reply:
I put this line in because I wanted to differentiate this page from the "About mongosync" page, but you are right that the title makes it redundant. I will take it out for now but let me know if you think there is a better way of summarizing this page is besides using the word "behavior" again.